CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 31.2
+- show memory profiler legend with proper line chart color or line style
```

Comment:
There was also a fix for the hot restart action with used in IntelliJ 2018.3.

Reply:
This was @pq's PR: https://github.com/flutter/flutter-intellij/pull/2895

Reply:
That PR was included in M31.1.